integration-tests/src/test/java/tachyon/client/FileOutStreamIntegrationTest.java

```diff
@@ -57,7 +58,9 @@ public static final void afterClass() throws Exception {
   @BeforeClass
   public static final void beforeClass() throws IOException {
     sLocalTachyonCluster = new LocalTachyonCluster(10000, 128, 128);
-    sLocalTachyonCluster.start();
+    TachyonConf tachyonConf = new TachyonConf();
+    tachyonConf.set(Constants.USER_FILE_BUFFER_BYTES, String.valueOf(BUFFER_BYTES));
```

Comment:
Other tests cases in the same file are "affected" since they will use the custom buffer size, but only the new test case I added fails (without the bug fix).